client/my-sites/email/email-providers-stacked-comparison/provider-cards/email-upsell-navigation.tsx

```diff
@@ -0,0 +1,24 @@
+import { Button, Gridicon } from '@automattic/components';
```

Comment:
It's a pity that this component did not exist before as part of the domain unification :( 

Probably as a follow up in the future we can ping Nomado and extend this component with actions/show/hide/labels and reuse it in their flow.